Update README.md (#823)
gta-reversed/gta-reversed: README.md

2) Select `conan-debug (conan-default-conan-debug)` or `conan-release (conan-default-conan-release)` (Depending on what you've spcified in the command above), other builds will result in CMake errors [TODO: Fix this somehow... We aren't proeficient with Conan/CMake sadly]
3) Alternatively you can type `cmake --preset conan-debug/release` and build with `cmake --build build` in CLI.

### 3. Set up the game and ASI
0. GTA SA ***Compact exe***: Our code requires you to use this exe, otherwise you will experience random crashes. You must own the original game and assets for this to work.

> 

Author: ```suggestion
[!IMPORTANT]
GTA SA ***Compact exe*** is not the same as ***1.0 US exe***, so check that your executable is exactly `5'189'632 bytes` (4.94 MiB).
```

### 3. Set up the game and ASI
0. GTA SA ***Compact exe***: Our code requires you to use this exe, otherwise you will experience random crashes. You must own the original game and assets for this to work.

> [!IMPORTANT]
> GTA SA ***Compact exe*** is not the same as ***1.0 US exe***, so check that your executable is exactly `5'189'632 bytes` (4.94 MiB).

Reviewer: ```suggestion
> [!IMPORTANT]
> GTA SA ***Compact exe*** is not the same as ***1.0 US exe***, so check that your executable is exactly `5'189'632 bytes` (4.94 MiB).
```

The DLLs it produces **do not work**.

### Game

> 1. GTA SA ***Compact exe***: Our code requires you to use this exe, otherwise you will experience random crashes. (Anything else **WILL NOT** work correctly!)
> 

Author: ```suggestion
[!IMPORTANT]
Follow these instructions closely, otherwise the game **will not** work as expected!

1. GTA SA ***Compact exe***: Our code requires you to use this exe, otherwise you will experience random crashes.
```
